rc2/rc2.go

```diff
@@ -0,0 +1,270 @@
+// Package rc2 implements the RC2 cipher
+/*
+https://www.ietf.org/rfc/rfc2268.txt
+http://people.csail.mit.edu/rivest/pubs/KRRR98.pdf
+
+This code is licensed under the MIT license.
```

Comment:
The rest of go-pkcs12 is also under MIT. I can of course relicense it under the Go license when that change occurs.